plugins/net_plugin/net_plugin.cpp

```diff
@@ -915,7 +843,8 @@ namespace eosio {
       std::atomic<uint16_t>   protocol_version = 0;
       uint16_t                net_version = net_version_max;
       std::atomic<uint16_t>   consecutive_immediate_connection_close = 0;
-      std::atomic<bool>       is_bp_connection = false;
+      std::atomic<bool>       is_gossip_bp_connection = false;
+      std::atomic<bool>       is_configured_bp_connection = false;
```

Comment:
Could these be replaced by an `enum class connection_type { non_bp, bp_config, bp_gossip };`?